packages/@atjson/renderer-html/src/index.ts

```diff
@@ -430,4 +432,53 @@ export default class HTMLRenderer extends Renderer {
         return `text-align:justify;`;
     }
   }
+
+  layoutStyle(layout: {
+    size?: "small" | "medium" | "large";
+    inset?: "left" | "right" | "center";
+    anchorName?: string;
+  }) {
+    if (layout.inset === "left") {
+      return "position-left";
+    } else if (layout.inset === "right") {
+      return "position-right";
+    } else if (layout.size === "small") {
+      return "size-small";
+    } else if (layout.size === "medium") {
+      return "size-medium";
+    } else if (layout.size === "large") {
+      return "size-large";
+    }
+    return undefined;
+  }
+
+  accordionSlicer(sliceId?: string) {
+    if (sliceId) {
+      let slice = this.getSlice(sliceId);
+      if (!slice) {
+        throw new Error(
+          `Accordion Header slice ${sliceId} not found in document`
+        );
+      }
+      let children = this.render(slice);
+      return `<accordion-${sliceId}>${children}</accordion-${sliceId}>`;
+    }
+    return "";
+  }
+
+  *AccordionGroup(accordionGroup: AccordionGroup) {
+    return yield* this.$("accordion-group", {
+      accordionGroupStyle: this.layoutStyle(accordionGroup.attributes),
+    });
+  }
```

Comment:
I do not think that there is a representation of the accordion group here, as we don't have HTML that would be created from another source (like, for example Word documents). So for now, I think removing this block renderer from here would be good 😄